packages/kbn-optimizer/src/worker/run_worker.ts

```diff
@@ -50,22 +50,6 @@ const send = (msg: WorkerMsg) => {
   }
 };
 
-/**
- * set the exitCode and wait for the process to exit, if it
- * doesn't exit naturally do so forcibly and fail.
- */
-const exit = (code: number) => {
-  process.exitCode = code;
-  setTimeout(() => {
-    send(
-      workerMsgs.error(
-        new Error('process did not automatically exit within 5 seconds, forcing exit')
```

Comment:
Should we still log an error?

Reply:
What error? If we call process.exit() the process is going to exit immediately and we won't be able to set a timer or anything.